docs/Advanced-Usage/Manager/Manager-Configuration-Files.rst

```diff
@@ -308,101 +309,174 @@ Here is a sample of this configuration file:
 
 Below, we outline each section and parameter in detail.
 
-
-``[afibuild]``
+``build_farm``
 ^^^^^^^^^^^^^^^^^^^^^
 
-This exposes options for AWS resources used in the process of building FireSim
-AGFIs (FPGA Images).
-
-``s3bucketname``
-""""""""""""""""""""""""""
-
-This is used behind the scenes in the AGFI creation process. You will only
-ever need to access this bucket manually if there is a failure in AGFI creation
-in Amazon's backend.
-
-Naming rules: this must be all lowercase and you should stick to letters and numbers.
+In this section, you specify the specific build farm name that you wish to use to build FPGA bitstreams (see ``config_build_farm.yaml`` for build farm options and specification).
```

Comment:
You might be able to provide the gh ref here to a sample?  Also "build farm name" -> "build farm definition"?

Reply:
I'll change this to instead refer to the doc section on build farms. Additionally, I think "build farm name" -> "build farm configuration" makes more sense here.